examples/build.gradle

```diff
@@ -3,7 +3,16 @@ plugins {
     id "com.github.johnrengelman.shadow" apply false
 }
 
-println("Building against OpenTelemetry version: ${project.properties["io.opentelemetry.version"]}")
```

Comment:
This always printed null, and a property which this name is nowhere to be found and not used anywhere else.

Reply:
yep, this was left over from when we could build against different versions with a provided property. Good catch!